card/mod/standard/set/all/rich_html/html_views/info.rb

```diff
@@ -37,7 +37,7 @@
   end
 
   view :type_info do
-    return unless show_view?(:toolbar, :hide) && card.type_code != :basic
+    return unless card.type_code != :basic
```

Comment:
fwiw, I think we should override this view in type/basic (but this is not new...)